data/adapters/helpers.py

```diff
@@ -30,3 +34,49 @@ def build_variant_id(chr, pos_first_ref_base, ref_seq, alt_seq, assembly='GRCh38
 @assembly_check
 def build_regulatory_region_id(class_name, chr, pos_start, pos_end, assembly='GRCh38'):
     return '{}_{}_{}_{}_{}'.format(class_name, chr, pos_start, pos_end, assembly)
+
+
+@assembly_check
+def build_variant_id_from_hgvs(hgvs_id, validate=True, assembly='GRCh38'):
+    # translate hgvs naming to vcf format e.g. NC_000003.12:g.183917980C>T -> 3_183917980_C_T
+    if validate:  # use tools from hgvs, which corrects ref allele if it's wrong
+        # got connection timed out error occasionally, could add a retry function
+        hdp = hgvs.dataproviders.uta.connect()
+        babelfish38 = Babelfish(hdp, assembly_name=assembly)
```

Comment:
we could generate a mapping file in the future of all translations so we don't depend on their service all the time